app/views/layouts/_navigation_links.html.erb

```diff
@@ -1,3 +1,7 @@
 <li class="">
+<% if current_user %>
+  <%= link_to "Sign Out", logout_path, class: 'navbar-brand' %>
+<% else %>
   <%= link_to 'Sign In With Slack', login_path('slack'), class: 'navbar-brand' %>
```

Comment:
If we use i18n for these, we won't have to worry about changing the wording and breaking tests.